### DISCUSSION TITLE
Welcome to jupytergis Discussions!

### DISCUSSION THREAD

**Discussion by martinRenou** [geojupyter/jupytergis#209] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mfisher87** [comment]
:wave: Thanks for getting this started, Martin :) I'm Matt, a research software engineer and Open Science & open source advocate/enthusiast. I'll be working on the early stages of [GeoJupyter in general](https://geojupyter.github.io/about.html), and am still learning about JupyterGIS and how I can effectively contribute.

To begin with, I'm going to be conducting [user needs interviews](https://geojupyter.github.io/interviews/). Please sign up if you're interested! We would love to hear your voice.

:heart: I'm so excited to be working with this community. Let's make some awesome stuff :)
